src/bin/cobalt/new.rs

```diff
@@ -175,20 +206,52 @@ pub fn create_new_document(
         )
     })?;
 
+    let pages = config.pages.clone().build()?;
     let posts = config.posts.clone().build()?;
-    let (file_type, doc) = if file.starts_with(posts.pages.subtree())
+    let file_type = if posts.pages.includes_file(&file)
         || posts
             .drafts
-            .as_ref()
-            .map(|d| file.starts_with(d.subtree()))
-            .unwrap_or(false)
+            .map(|d| d.includes_file(&file))
+            .unwrap_or_default()
+    {
+        posts.slug.as_str()
+    } else if pages.pages.includes_file(&file)
+        || pages
+            .drafts
+            .map(|d| d.includes_file(&file))
+            .unwrap_or_default()
     {
-        ("post", POST_MD)
+        pages.slug.as_str()
+    } else {
+        bail!(
+            "Target file wouldn't be a member of any collection: {:?}",
+            file
+        );
+    };
+
+    let source_path = config
+        .source
+        .join(format!("_defaults/{}.{}", file_type, extension));
```

Comment:
As I said elsewhere, I hate this directory name.  Any other thoughts?

Reply:
I propose `_default_templates`. It's a bit long, but its purpose can be inferred without looking at any documentation.

Reply:
This is though, I like gutenberg's choice for it, a bit obscure at first, but archetype names very well what's inside this folder. It will teach a word to people ;)

Reply:
Gutenberg copied Hugo's use of archetype?  I couldn't find anything in the gutenberg docs related to this.

I forgot another option I thought of is "_prototypes" like the Prototype Pattern or Javascript Prototypes.  The word, at least among programmers, is more common than archetype and the connotation, at least to programmers, is slightly more accurate.

Reply:
Sorry, I made a mistake, I meant Hugo ^^'
_prototypes is not bad :)